packages/next/kotti-field-radio-group/KtFieldRadioGroup.vue

```diff
@@ -60,6 +60,9 @@ export default defineComponent({
 				[...new Set(options.map(({ value }) => value))].length ===
 				options.length,
 		},
+		isInline: {
+			type: Boolean,
+		},
```

Comment:
`default: false` is missing. Booleans should always `default` to `false`.
```suggestion
		isInline: { default: false, type: Boolean },
```

Also, this should be sorted above `options`